### DISCUSSION TITLE
headless install

### DISCUSSION THREAD

**Discussion by gildas-declercq** [bareboat-necessities/lysmarine_gen#354] [General]
Hi all,

when I write on SD card a new install, and run it without screen the lysmarine-hotsport does not appear.
I boot again with a screen, hotspot is ok and I can uncomment 
hdmi_force_hotplug=1
/boot/config.txt to run without screen.

how can I have the hotspot from the first boot without screen attached ?
I tried to modify the config.txt directly on the SD card but without success.

thanx a lot
Gildas

**Comment by mgrouch** [comment]
hotspot is created by NetworkManager.

https://github.com/NetworkManager/NetworkManager

It's software Debian started using in bookworm as default. 

I’m surprised that it would depend on GUI being up. It should start regardless if GUI is up or not. I’d need some time to reproduce and investigate

**Reply by gildas-declercq** [reply]
ok thank you. I am 99% sure , as I did it a couple of times.
here are my tests

power the pi with the newly burnt SD card (2023 08  17 version)
I waited several minutes.
I cut the power, then power again. 
I cut the power, connect hdmi, power => I see it booting and the hotspot is on.

**Comment by mgrouch** [comment]
Try asking on NetworkManager issues

https://gitlab.freedesktop.org/NetworkManager/NetworkManager/-/issues

I do not know if BBN sets anything specific to NetworkManager services and hotspot requiring GUI

**Comment by mgrouch** [comment]
Have you set up WiFi country?

**Reply by gildas-declercq** [reply]
no because the idea is to get access to the BBN at first boot.

I have no screen on my boat, and I would like to be able to install a BBN from scratch.
I don't have wire network neither for now.
I can of course modify the files on the SD card when accessible from my computer

**Reply by mgrouch** [reply]
Actually in that version WiFi country is already defaulted to US. So probably country is not the cause of the issue.

**Reply by mgrouch** [reply]
There is a thing for $4 called 
“Dummy HDMI plug”. Might help to solve this issue

**Reply by gildas-declercq** [reply]
it is really an issue, 
more like I would understand where it is failing to turn on hotspot.
it is also to learn rpi / BBN stuff.

as openplotter "headless" image can do it guess it is a parameter somewhere.

thanx anyway

**Reply by mgrouch** [reply]
OP is not using NetworkManager for hotspot
